db/seeds.rb

```diff
@@ -0,0 +1,52 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
+#
+# Examples:
+#
+#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
+#   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Post.create({
+  title: "Post Title 1",
+  body: "This is body #1"
+  })
+
+Comment.create({
+  body: "Here's my comment 1",
+  post_id: 1
+  })
+
+Comment.create({
+  body: "Here's my comment 2",
+  post_id: 1
+  })
+
+Post.create({
+  title: "Post Title 2",
+  body: "This is body #2"
+  })
+
+Comment.create({
+  body: "Here's my comment 1",
+  post_id: 2
+  })
+
+Comment.create({
+  body: "Here's my comment 2",
+  post_id: 2
+  })
+
+Post.create({
+  title: "Post Title 3",
+  body: "This is body #3"
+  })
+
+Comment.create({
+  body: "Here's my comment 1",
+  post_id: 2
+  })
+
+Comment.create({
+  body: "Here's my comment 2",
+  post_id: 2
+  })
```

Comment:
The trouble with this is that it's assuming what each post's ID will be. If you run the seed file more than once, this will be inaccurate. I recommend saving each post to a variable and then doing `@post1.comments.create()`